_pages/robot/2018.html

```diff
@@ -7,8 +7,8 @@
 ---
 <main>
     <h1>2018 Robot</h1>
-    <img src="/assets/img/robots/2018/redone.jpg" alt="Our current robot (front view)."></img>
-    <img src="/assets/img/robots/2018/redtwo.jpg" alt="Our current robot (side view)."></img>
+    <img src="/assets/img/robots/2018/ROBONE.JPG" alt="Our current robot (front view)."></img>
+    <img src="/assets/img/robots/2018/ROBTWO.JPG" alt="Our current robot (side view)."></img>
```

Comment:
There's not much difference between these two photos. I'd be okay with just having the second one for now, and adding a different photo later on.